CHANGELOG.md

```diff
@@ -0,0 +1,19 @@
+# v2.1.0 (2024-07-08)
```

Comment:
nit: Ideally we would have one commit for the CHANGELOG entry and one for version bumps.